src/store/mesh/getters.ts

```diff
@@ -61,24 +68,42 @@ export const getters: GetterTree<MeshState, RootState> = {
       profiles.push({
         name,
         active: name === bedMesh.profile_name,
-        min,
-        max,
```

Comment:
Shouldn't `min` and `max` be kept here?

Reply:
It's not shown in the bed mesh profile list, but I think there's no harm in keeping it. I'll add those back later

Reply:
That's a very good point: I did a quick check, and I can't find it in use anywhere, so I see no reason why we can't just drop those properties!